helium/src/scss/blocks/_course_list.scss

```diff
@@ -2,7 +2,9 @@
 @import "ycthemelib/responsive";
 @import "constants";
 
-.yc-course-list {
+// retain use of course_list class only for backwards compatibility with any
+// academies/course with custom css
+.yc-course-list, .course_list {
```

Comment:
This will not ensure backwards compatibility - the custom css will expect to have a class `.course_list`. The template should be updated to have this class.